src/Handlers/ResolveHandlers.php

```diff
@@ -164,7 +164,14 @@ protected function continueConversation(Conversation|callable $conversation): ar
     {
         $resolvedHandlers = [];
 
-        if (!$conversation instanceof Conversation || !$conversation->skipHandlers()) {
+        /** @var bool|UpdateType[] $skipHandlers */
+        $skipHandlers = $conversation instanceof Conversation ? $conversation->skipHandlers() : false;
+        if (is_array($skipHandlers)) {
+            $updateType = $this->update?->getType();
+            $skipHandlers = in_array($updateType, $skipHandlers, true);
+        }
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common processes. Try to leave a link to the process described so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏This logic should be likely contained inside the skipHandlers() method instead of the resolver, since is conversation business know if should skip or not.